frontend/packages/dev-console/src/components/import/toast/ImportToastContent.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import RouteLinkAndCopy from '@console/internal/components/routes';
+import { K8sResourceKind, RouteKind } from '@console/internal/module/k8s';
+
+interface ImportToastContentProps {
+  deployedResources: K8sResourceKind[];
+  route?: RouteKind;
+}
+
+const ImportToastContent: React.FC<ImportToastContentProps> = ({
+  deployedResources = [],
+  route,
+}) => {
+  const { t } = useTranslation();
+  if (!deployedResources || deployedResources.length === 0) {
+    return null;
+  }
+  return (
+    <>
+      {deployedResources.length > 1
+        ? t('devconsole~Deployments created successfully.')
+        : t('devconsole~Deployment created successfully.')}
+      {route && <RouteLinkAndCopy route={route} />}
```

Comment:
Sorry, but this looks also still unaligned. The link right behind the message without any spacing? I would suggest a new paragraph with some vertical spacing between the message and the link here.

![unaligned-content](https://user-images.githubusercontent.com/139310/141473391-879b4268-4871-47cc-accf-72ebe8856b82.png)